### PR TITLE
Reflect changes in Model to the ModelManager API

### LINKED ISSUE #81: Reflect changes in Model to the ModelManager API
The Tag model has been modified, and a UniqueTagList class has been added to accommodate new features.
However, these additions to the model cannot be used until its methods are exposed through ProjactManager class.

Task:
- Allow the methods of UniqueTagList to be called from ProjactManager class

### PR DESCRIPTION
**Pull request by akgrenSoar**
Closes #81

### CODE REVIEW

**Review by siqiang-ng — COMMENTED** [review]
LGTM

File: src/main/java/seedu/address/model/ModelManager.java [hunk 1]
```diff
@@ -27,7 +28,8 @@
     private final Projact projact;
     private final UserPrefs userPrefs;
     private final FilteredList<Person> filteredPersons;
-    private Predicate<TagName> filteredTagPredicate = x -> true;
+    private final FilteredList<Tag> filteredTags;
+    private Predicate<TagName> filteredTagPredicate = x -> true; // TODO: Remove in v1.3
```
Comment (yeo-kexin) [inline src/main/java/seedu/address/model/ModelManager.java:32]: so just to clarify, this would be done by kk right?

**Review by yeo-kexin — COMMENTED** [review]
LGTM

**Review by khinkhinn — COMMENTED** [review]
lgtm

File: src/main/java/seedu/address/model/ModelManager.java (hunk 1 above)
Comment (khinkhinn) [reply src/main/java/seedu/address/model/ModelManager.java:32]: sorry isit??? i dont think so..? do correct me if im wrong..!!

**Review by JoeyToh — APPROVED** [review]
lgtm

**Pull request merged**